src/index.js

```diff
@@ -0,0 +1,199 @@
+    state.tempValue.classList.add('green');
+  } else if (state.temperature <= 49) {
+    state.tempValue.classList.add('teal');
+  }
+  //TOGGLE LANDSCAPE
+  state.tree[0].classList.toggle('show', state.temperature <= 49);
+  state.garden.classList.toggle('snowy-sky', state.temperature <= 49);
+  state.ground[0].classList.toggle('snowy', state.temperature <= 49);
+
+  state.tree[2].classList.toggle('show', state.temperature >= 50 && state.temperature <= 64);
+  state.ground[0].classList.toggle('cloudy', state.temperature >= 50 && state.temperature <= 64);
+  state.garden.classList.toggle('cloudy-sky', state.temperature >= 50 && state.temperature <= 64);
+
+  state.tree[1].classList.toggle('show', state.temperature >= 65 && state.temperature <= 75);
+
+  state.tree[3].classList.toggle('show', state.temperature > 75);
+  state.ground[0].classList.toggle('sunny', state.temperature > 75);
+};
+
+const changeSkyscape = () => {
+  state.sky[0].textContent = '';
+  // CLOUDY
+  if (state.skySelect.value === 'cloudy') {
+    state.sky[0].textContent = '🌾🍃🌾';
+  }
+  // SUNNY
+  if (state.skySelect.value === 'sunny') {
+    state.sky[0].textContent = '☀️😎☀️';
+  }
+
+  // SNOWY
+  if (state.skySelect.value === 'snowy') {
+    state.sky[0].textContent = '❄️🥶❄️';
+  }
+
+  // RAINY
+  if (state.skySelect.value === 'rainy') {
+    state.sky[0].textContent = '💧☔️💧';
+  }
+
+  //CLOUDY or RAINY
+  state.cloud[0].classList.toggle('show', state.skySelect.value === 'cloudy' || state.skySelect.value === 'rainy');
+  state.cloud[1].classList.toggle('show', state.skySelect.value === 'cloudy' || state.skySelect.value === 'rainy');
+};
+
+const changeCityName = () => {
+  state.headerCityName.textContent = state.input.value;
+};
+
+const resetCity = () => {
+  state.input.value = '';
+  state.headerCityName.textContent = 'Primm';
+  updateWithRealtimeWeather()
+};
+
+const parseRegion = (region) => {
+  region = region.split(',');
+  region = `${region[0]}, ${region[region.length - 2]}, ${region[region.length - 1]}`;
+  return region;
+};
+
+const getLatLon = async () => {
+  const response = await axios.get('https://weather-report-proxy-msis.onrender.com/location', { params: { q: state.headerCityName.textContent } });
+  const { lat: latitude, lon: longitude } = response.data[0];
+  state.headerCityName.textContent = parseRegion(response.data[0].display_name);
+  return { latitude, longitude };
+};
+
+const setSkyWithRealtimeWeather = (code) => {
+  code = code.toString();
+  if (code.match(/2../) || code.match(/3../) || code.match(/5../)) {
+    state.skySelect.value = 'rainy';
+  }
+  else if (code.match(/6../)) {
+    state.skySelect.value = 'snowy';
+  }
+  else if (code.match(800)) {
+    state.skySelect.value = 'sunny';
+  }
+  else {
+    state.skySelect.value = 'cloudy';
+  }
+  changeSkyscape();
+};
+
+const getRealtimeWeather = async () => {
+  const { latitude, longitude } = await getLatLon();
```

Comment:
Rather than calling the location function directly, prefer making a pure weather function that accepts a location in the parameters. We could then make a _third_ function that takes care of orchestrating the use of the two functions (location and weather) together.